pvlib/test/test_singlediode_methods.py

```diff
@@ -0,0 +1,172 @@
+"""
+testing single-diode methods using JW Bishop 1988
+"""
+
+from time import clock
+import logging
+import numpy as np
+from pvlib import pvsystem
+from conftest import requires_scipy
+
+logging.basicConfig()
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.DEBUG)
+
+POA = 888
+TCELL = 55
+CECMOD = pvsystem.retrieve_sam('cecmod')
+
+
+@requires_scipy
+def test_fast_spr_e20_327():
```

Comment:
these function names need to be more descriptive.

Reply:
✔️